src/main/java/com/ncedu/cheetahtest/service/testscenario/TestScenarioServiceImpl.java

```diff
@@ -60,46 +72,37 @@ public TestScenario createTestScenario(TestScenario testScenario, List<ActionsAn
         }
 
         testScenario.setStatus(ACTIVE);
+        TestScenario createdTestScen;
         createdTestScen = testScenarioDao.createTestScenario(testScenario);
 
         ActScenario actScenario = new ActScenario();
 
         int priority = 1;
-        for (ActionsAndCompoundsID actAndComp: actAndCompID) {
+        for (ActionsAndCompoundsID actAndComp : actAndCompID) {
             if (actAndComp.getKind().equals("ACTION")){
```

Comment:
Will be better to use enum for actAndComp kind

Reply:
it is safe anyway